src/components/views/Header.js

```diff
@@ -12,7 +12,7 @@ import "styles/views/Header.scss";
  */
 const Header = (props) => (
   <div className="header container" style={{ height: props.height }}>
-    <h1 className="header title">SoPra FS22 rocks with React!</h1>
+    <p className="header title">Brändi Dog</p>
```

Comment:
```suggestion
    <h1 className="header title">Brändi Dog</h1>
```
SEO wise, a heading makes more sence here :) Did you chose `<p>` because `<h1>` is already styled?